### PR TITLE
feat: Image support in editor

### PR DESCRIPTION
**Pull request by iamsivin**

### CODE REVIEW

File: src/schema/markdown/messageParser.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ export const messageSchemaToMdMapping = {
...
   ...baseNodesMdToPmMapping,
   ...baseMarksMdToPmMapping,
+  image: {
+    node: 'image',
+    getAttrs: (tok) => ({
+      src: tok.attrGet('src'),
+      title: tok.attrGet('title') || null,
+      alt: (tok.children[0] && tok.children[0].content) || null,
+    }),
+  },
```
Comment (pranavrajs) [inline src/schema/markdown/messageParser.js]: Didn't we already support this in article editor?

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please see the comment.

File: src/schema/markdown/messageParser.js (hunk 1 above)
Comment (iamsivin) [reply src/schema/markdown/messageParser.js]: @pranavrajs Yes we support it in article editor. But in our message editor we are using this message parser that not supports images.

Comment (pranavrajs) [reply src/schema/markdown/messageParser.js]: Can't we combine this similar to the buildEditor method instead of making it as a 2 different config?

Comment (iamsivin) [reply src/schema/markdown/messageParser.js]: @pranavrajs Done https://github.com/chatwoot/prosemirror-schema/pull/17/commits/825755b53f1ef4ae14fac2393b53a02e14ce21e0

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**